modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -4,21 +4,17 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: runtime, gateway, gatekeeper, security, policy, policies
 
-When you deploy an application, there could be a lapse during which the tracked resources exposed by the application are accessible, but the policies specified for those tracked resources are not yet applied, so unwelcome traffic may reach the them. GateKeeper prevents this situation by blocking the tracked resource until all policies have been retrieved and applied without errors. During this period, the API returns a 503 HTTP status code.
+When you deploy an application, the tracked resources exposed by the application might become accessible due to a lapse. Because the policies specified for those tracked resources are not yet applied, the API might receive unwanted traffic. GateKeeper prevents this situation by blocking the tracked resource until all policies are retrieved and applied without errors. 
```

Comment:
due to a lapse. Seems that the sentence is incomplete

Reply:
Done! Thanks!